crypto/crypto.go

```diff
@@ -257,11 +267,31 @@ func DecodePublicKeyPEM(sigAlgo SignatureAlgorithm, s string) (PublicKey, error)
 	}
 
 	goPublicKey := publicKey.(*ecdsa.PublicKey)
-
-	rawPublicKey := append(
-		goPublicKey.X.Bytes(),
-		goPublicKey.Y.Bytes()...,
-	)
+	xBytes := goPublicKey.X.Bytes()
+	yBytes := goPublicKey.Y.Bytes()
+	expectedLength := expectedPointByteLength[sigAlgo]
```

Comment:
It's better to get that length this way:
```suggestion
	expectedLength := bitsToBytes(goPublicKey.Params().P.BitLen())
```
With `bitsToBytes` the function converting bit size into byte size:
```
func bitsToBytes(bits int) int {
	return (bits + 7) >> 3
}
```